package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "react-resource-router",
-  "version": "0.28.0",
+  "version": "0.29.0",
```

Comment:
Missing `package.json` change?

Reply:
Package.json is already at "version": "0.29.0",

Reply:
I guess this probably means changesets don't run the package manager correctly on bump, I suppose that's fine since it's local to the repo